src/CoreBundle/private/library/classes/TGoogleMap/TGoogleMapEndPoint.class.php

```diff
@@ -218,25 +221,52 @@ public function hookMenueLinks($b)
      *
      * @param string $sPlace
      *
-     * @return array|bool - contains latitude and longitude coordinate
+     * @return array{latitude: float, longitude: float}|false - contains latitude and longitude coordinate
      */
     public function locatePlace($sPlace = '')
     {
-        $aCoordinates = false;
-        $sURL = 'http://maps.googleapis.com/maps/api/geocode/json?address='.urlencode($sPlace);
-
-        $sResponse = file_get_contents($sURL);
-        if (empty($sResponse)) {
+        $url = sprintf('https://nominatim.openstreetmap.org/search?%s', http_build_query([
```

Comment:
Fair enough. I made the endpoint configurable and changed the frontend code from calling the endpoint directly to calling a controller, which then does geocoding from PHP. This was done in order to have consistent geocoding behaviour in case we want to replace geocoding behaviour with something else